vignettes/GOvis.Rmd

```diff
@@ -0,0 +1,547 @@
+---
+title: "Visualizing and Distances Using GO"
+authors:
+  - name: "R. Gentleman"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "`r format(Sys.time(), '%B %d , %Y')`"
+vignette: >
+  %\VignetteIndexEntry{Visualizing and Distances Using GO}
+  %\VignetteDepends{Biobase,genefilter,graph,GO.db,hgu95av2.db,xtable,multtest}
+  %\VignetteSuggests{Rgraphviz}
+  %\VignetteKeywords{EDA, graphs, ontology, visualization}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignettePackage{GOstats}
+output:
+  BiocStyle::html_document
+bibliography: GOstats.bib
+link-citations: yes
+---
+
+# Introduction {#introduction .unnumbered}
+
+The basic characteristics of the GO [@GO]
+data are described in @GO and the
+interested reader is referred there for more details. In this
+paper we assume
+that readers are familiar with the basic DAG structure of GO
+and with the
+mappings of genes to GO terms that are provide by GOA [@GOA].
+We examine 
+some of the different visualization problems and data
+analytic problems that can be addressed using these data. To
+make some of the
+concepts concrete and to provide extensive, but related
+examples we will make
+use of a microarray data set [@Chiaretti2004].
```

Comment:
Can I ask you to reformat sections that are like these so that they are easier to read?

Reply:
Ohk I will do that for all the paragraphs.